x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/transport/RemoteClusterCredentialsResolver.java

```diff
@@ -27,11 +27,6 @@ public class RemoteClusterCredentialsResolver {
 
     private final Map<String, String> apiKeys = ConcurrentCollections.newConcurrentMap();
 
-    /**
-     * Initialize load and reload REMOTE_CLUSTER_AUTHORIZATION values.
-     * @param settings Contains zero, one, or many values of REMOTE_CLUSTER_AUTHORIZATION literal values.
-     * @param clusterSettings Contains one affix setting REMOTE_CLUSTER_AUTHORIZATION.
-     */
     public RemoteClusterCredentialsResolver(final Settings settings, final ClusterSettings clusterSettings) {
```

Comment:
Sticking with `RemoteCluster...` here instead of `CrossClusterAccess...` because it's closer to the setting name (and in the querying cluster side view, `RemoteCluster...` is sensible here).

I will rename the `REMOTE_CLUSTER_AUTHORIZATION` setting to something ending in credentials in a follow-up because there might be a version dance we need to do there.